library/src/main/java/com/wultra/android/mtokensdk/api/operation/model/OperationAttribute.kt

```diff
@@ -23,7 +23,7 @@ import java.math.BigDecimal
  * Operation Attribute can be visualized as "1 row in operation screen"
  * Every type of the attribute has it's own strongly typed implementation based on its [type]
  */
-sealed class Attribute(
+open class Attribute(
```

Comment:
Reason? Do we allow custom extension attributes?

Reply:
No, but for the `UNKNOWN` type it's now possible to instantiate just the `Attribute` so it cannot be sealed anymore.